### PR TITLE
feat: added basic wordpress docs

### PR DESCRIPTION
**Pull request by markusahlstrand**
Added very rough documentation for the wordpress plugin

### CODE REVIEW

File: docs/integration/cms/wordpress/passes.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+---
+sidebar_position: 3
+---
+
+# Passes
+
+A pass is used to provide access to multiple articles over a time period, typically all locked content for for instance a month.
```
Comment (julianidiart) [inline docs/integration/cms/wordpress/passes.md:7]: for, for instance, a month

File: docs/integration/cms/wordpress/passes.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+The passes can be defined direclty in wordpress.
```
Comment (julianidiart) [inline docs/integration/cms/wordpress/passes.md:9]: Typo: `directly`

**Review by julianidiart — APPROVED** [review]
Looks good, with only some typos. Makes sense as to how it's implemented to me, but cannot confirm since I haven't tried it, maybe Jonas can confirm that the text is correct?

**Comment by markusahlstrand** [comment]
Thanks for the review!

**Pull request merged**